src/adv.py

```diff
@@ -49,3 +50,34 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+# for singleRoom in room:
+#     print(f'{singleRoom}: {room[singleRoom].name}')
+while True:
+    currentRoom = player.getRoom()
+
+    print(currentRoom)
+    # print(f"Current room: {currentRoom}")
+    # print(getattr(room[currentRoom], 'n_to'))
+    # print(currentRoom.n_to)
+
+    userInput = input('Enter where to go: ')
+    if(userInput == 'n'):
+        # print(room[currentRoom].n_to)
+
+        player.setRoom(currentRoom.n_to)
+    elif (userInput == 's'):
+        player.setRoom(currentRoom.n_to)
+
+    elif (userInput == 'w'):
+        player.setRoom(currentRoom.w_to)
+    elif (userInput == 'e'):
+        player.setRoom(currentRoom.e_to)
```

Comment:
Can we make it more dynamic? Maybe by using Fstring for calling current_room methods.